Gemfile

```diff
@@ -43,6 +43,7 @@ group :development, :test do
   gem "dotenv-rails"
   gem "letter_opener"
   gem "i18n-tasks"
+  gem "erb_lint"
```

Comment:
What do you think of adding a `.erb-lint.yml` file for a bit more explicitness and integrating with our Rubocop setup?

```yml
EnableDefaultLinters: true
linters:
  Rubocop:
    enabled: true
    rubocop_config:
      inherit_from:
        - .rubocop.yml
```

Reply:
@zachgoll Yes, I forget about the config for the `erblint`. At the same time, I am completely against enabling all Rubocop rules for `erblint`. I tried doing this locally, and I got about 3500 errors with only 3000 auto-correctable. Besides this, some HTML tags started to look really strange after the automatic correction.

Reply:
I added `.erb-lint.yml`, please check.

Reply:
Sounds good, and very understandable on the rubocop integration.  I'm all for auto checks, but not if it's going to slow us down too much.